provisioning/roles/solr-configuration/templates/solr_7.x_config-drupal8/solrcore.properties

```diff
@@ -4,11 +4,10 @@ solr.replication.pollInterval=00:00:60
 solr.replication.masterUrl=http://localhost:8983/solr
 solr.replication.confFiles=schema.xml,schema_extra_types.xml,schema_extra_fields.xml,elevate.xml,stopwords_en.txt,synonyms_en.txt,protwords_en.txt,accents_en.txt,stopwords_und.txt,synonyms_und.txt,protwords_und.txt,accents_und.txt
 solr.mlt.timeAllowed=2000
-solr.luceneMatchVersion=7.0
 solr.selectSearchHandler.timeAllowed=-1
 solr.autoCommit.MaxDocs=-1
 solr.autoCommit.MaxTime=15000
 solr.autoSoftCommit.MaxDocs=-1
 solr.autoSoftCommit.MaxTime=-1
-# Do not set install.dir. https://www.drupal.org/project/search_api_solr/issues/3015993
-#solr.install.dir=../../..
+
+solr.luceneMatchVersion=7.7
```

Comment:
This may possibly be the only line required to clear up the error I was experiencing.